_studio/shared/include/mfx_config.h

```diff
@@ -123,6 +123,15 @@
     #endif
 #endif
 
+#if !defined(ANDROID)
+#define MFX_ENABLE_HEVCEHW_REFACTORING
+#endif //!defined(ANDROID)
+
+#if defined(MFX_ENABLE_HEVCEHW_REFACTORING)
+#define MFX_ENABLE_HEVCEHW_REFACTORING_WIN_SKL
```

Comment:
This will enable SKL only or it will enable gen9+ support?

Reply:
Only gen9 graphics, not only SKL

Reply:
In this case this macro name is wrong and misleading. Please, rename to MFX_ENABLE_HEVCEHW_REFACTORING_WIN_GEN9 and MFX_ENABLE_HEVCEHW_REFACTORING_LIN_GEN9 first.

Secondly, why we switch new code for GEN9 only?

Reply:
it is agreement with HEVCe scrum team to make switch platform by platform. now SKL has acceptable passrates and all perf, stress, quality data so it is ready for switch

Reply:
ok